src/physics_swe.c

```diff
@@ -150,7 +187,8 @@ static PetscErrorCode RDyCeedOperatorSetUp(RDy rdy) {
         CeedOperatorSetField(op, "cell_left", restrict_l, CEED_BASIS_COLLOCATED, CEED_VECTOR_ACTIVE);
         CeedOperatorSetField(op, "cell_right", restrict_r, CEED_BASIS_COLLOCATED, CEED_VECTOR_ACTIVE);
         CeedOperatorSetNumQuadraturePoints(op, 1);
```

Comment:
@jedbrown It seems that `CeedOperatorSetNumQuadraturePoints` should not be called with the latest libCEED version as I'm getting the following error:

```
/Users/bish218/projects/petsc/petsc_99e66fd434959f7d2/clang-openmpi-libceed-1ca28fe1/externalpackages/git.libceed/interface/ceed-operator.c:1054 in CeedOperatorSetNumQuadraturePoints(): Number of quadrature points already defined
[WD86346:20769] *** Process received signal ***
[WD86346:20769] Signal: Abort trap: 6 (6)
[WD86346:20769] Signal code:  (0)
[WD86346:20769] [ 0] 0   libsystem_platform.dylib            0x0000000198c3aa24 _sigtramp + 56
[WD86346:20769] [ 1] 0   libsystem_pthread.dylib             0x0000000198c0bc28 pthread_kill + 288
[WD86346:20769] [ 2] 0   libsystem_c.dylib                   0x0000000198b19ae8 abort + 180
[WD86346:20769] [ 3] 0   libceed.dylib                       0x00000001024d8840 CeedAddJitSourceRoot + 0
[WD86346:20769] [ 4] 0   libceed.dylib                       0x00000001024d64b4 CeedErrorImpl + 108
[WD86346:20769] [ 5] 0   libceed.dylib                       0x00000001024bc118 CeedOperatorSetNumQuadraturePoints + 180
[WD86346:20769] [ 6] 0   rdycore                             0x00000001008cead0 RDyCeedOperatorSetUp + 2300
[WD86346:20769] [ 7] 0   rdycore                             0x00000001008c92f0 RDyCeedOperatorApply + 400
[WD86346:20769] [ 8] 0   rdycore                             0x00000001008c8608 RHSFunctionSWE + 1568
[WD86346:20769] [ 9] 0   libpetsc.3.019.1.dylib              0x0000000108f6bc24 TSComputeRHSFunction + 4248
[WD86346:20769] [10] 0   libpetsc.3.019.1.dylib              0x0000000108f2c5c8 TSStep_Euler + 632
[WD86346:20769] [11] 0   libpetsc.3.019.1.dylib              0x0000000108fb3b40 TSStep + 3040
[WD86346:20769] [12] 0   libpetsc.3.019.1.dylib              0x0000000108fbe6e0 TSSolve + 11248
[WD86346:20769] [13] 0   rdycore                             0x00000001008f4c80 RDyAdvance + 6248
[WD86346:20769] [14] 0   rdycore                             0x00000001008c0690 main + 1688
[WD86346:20769] [15] 0   dyld                                0x00000001988b3f28 start + 2236
```

Deleting `CeedOperatorSetNumQuadraturePoints` results in no error, but I  want to confirm that this line should be deleted.

Reply:
Yeah, that function is unnecessary now with some internal changes

Reply:
@bishtgautam , have you removed the call to this function and verified that it works with the latest CEED? Do we need to update our PETSc Docker setup? Or is this for a future PR?

Reply:
I have verified that we don't need this function with the latest CEED, but I'm not yet ready to switch to the latest version of PETSc because (1) there are a few other calls to PETSc functions that will also have to be updated and (2) I will also have to update my PETSc installation across multiple machines. So, we will update our code to be compatible with the latest libCEED+PETSc version in a future PR.

Reply:
We are updating to a warning instead of an error here when using `CeedOperatorSetNumQuadraturePoints`, but this function will be removed in the next libCEED release (not needed anymore anyways)

https://github.com/CEED/libCEED/pull/1293

Reply:
@jeremylt Thanks for the heads up.